recipe/meta.yaml

```diff
@@ -28,6 +29,7 @@ test:
   imports:
     - napari_matplotlib
   commands:
+    - "python -c 'from napari_matplotlib._version import version;print(version)'"
```

Comment:
```suggestion
    - "python -c 'from napari_matplotlib import __version__;print(__version__)'"
```
So it's testing public API?

Reply:
Yep that also works, thanks :) 

When using `pip list` I was getting a 0.0.0 version which is affecting another pip check on a different PR that depended on this one.